addons/payment/models/payment_provider.py

```diff
@@ -247,6 +247,20 @@ def _compute_feature_support_fields(self):
 
     #=== ONCHANGE METHODS ===#
 
+    @api.constrains('capture_manually')
+    def _check_capture_manually(self):
+        if self.capture_manually:
+            unsupported_pms = self.payment_method_ids.filtered(
+                lambda method: method.support_manual_capture == 'none' and method.active
+            ).mapped('name')
```

Comment:
To work with records and avoid wrongly using the term "unsupported":
```suggestion
            incompatible_pms = self.payment_method_ids.filtered(
                lambda method: method.support_manual_capture == 'none' and method.active
            )
```